Gemfile.lock

```diff
@@ -305,7 +307,7 @@ DEPENDENCIES
   webmock (~> 3.0)
 
 RUBY VERSION
-   ruby 2.4.1p111
+  ruby 2.4.1p111
 
 BUNDLED WITH
-   1.15.4
+  1.15.4
```

Comment:
Same as above–why was the spacing changed here?

Reply:
Indentation fixing, same as above.